src/main/java/module/teamAnalyzer/ui/RecapPanelTableModel.java

```diff
@@ -334,6 +334,7 @@ private String formatTacticSkill(TeamLineup lineup) {
 
     public TeamLineup getTeamMatchReport(int minSelectionIndex) {
         if ( teamReport != null){
+            var table = getTable();
```

Comment:
Is this used at all?

Reply:
Thanks a lot for your review!

the table is used in the next line to examine the index in the data model.